HW_6/main.py

```diff
@@ -0,0 +1,205 @@
+from datetime import datetime
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel, Field
+from sqlalchemy import create_engine, MetaData, Table, Column, Integer, String, Float, Date
+from databases import Database
+import uvicorn
+
+DATABASE_URL = "sqlite:///./test.db"
+
+database = Database(DATABASE_URL)
+
+metadata = MetaData()
+
+
+clients = Table(
+    "client",
+    metadata,
+    Column("id", Integer, primary_key=True),
+    Column("firstname", String(50)),
+    Column("lastname", String(50)),
+    Column("email", String(128))
+)
+
+
+products = Table(
+    "product",
+    metadata,
+    Column("id", Integer, primary_key=True),
+    Column("name", String(50)),
+    Column("description", String(255)),
+    Column("price", Float)
+)
+
+
+orders = Table(
+    "order",
+    metadata,
+    Column("id", Integer, primary_key=True),
+    Column("id_client", Integer, foreign_key=clients.c.id),
+    Column("id_product", Integer, foreign_key=products.c.id),
+    Column("date_order", Date),
+    Column("status", String, default="Создан")
+)
+
+engine = create_engine(DATABASE_URL)
+
+metadata.create_all(engine)
+
+
+class ClientIn(BaseModel):
+    firstname: str = Field(..., max_length=50)
+    lastname: str = Field(max_length=50)
+    email: str = Field(..., max_length=128)
+
+
+class Client(BaseModel):
+    id: int
+    firstname: str = Field(..., max_length=50)
+    lastname: str = Field(max_length=50)
+    email: str = Field(..., max_length=128)
```

Comment:
А тут можно заметить, что классы практически идентичны, а значит, один может наследоваться от другого.
```suggestion
class Client(ClientIn):
    id: int
```
И также сделать с остальными.